.github/ISSUE_TEMPLATE/package.yml

```diff
@@ -0,0 +1,41 @@
+name: Packages
+description: Report problems and suggest ideas for the packages
+labels: [package]
+
+body:
+  - type: markdown
+    attributes:
+      value: Before opening a new issue, please search for duplicate or closed issues.
+
+  - type: dropdown
+    attributes:
+      label: Kind of issue
+      description: Skip to "other" if you do not see your issue type here.
+      options: [improvement, bug]
+
+  - type: input
+    attributes:
+      label: Kind of issue (other)
+      description: Only fill out if your issue type is not in the dropdown above.
+
+  - type: dropdown
+    attributes:
+      label: Which package does this issue concern?
+      options: [NPM, Packagist]
+      multiple: true
+    validations:
+      required: true
+
+  - type: textarea
+    attributes:
+      label: Description
+      description: |
+        Anything relevant, for example:
+        - For bugs: "Steps to reproduce" and "Expected behavior"
+        - For improvements: An example of a use case
+        - Package issues: Version information
+          - For JavaScript/NodeJS: $ node --version
+          - For PHP/Packagist: $ composer --version
+        - etc.
```

Comment:
I wonder if we should convert these into separate (not required) inputs - though I'm not quite sure what that looks like once the form is submitted :thinking: Do you know of any project already using these that we can see? Or else, can you get these changes into your forks default branch so we can test it with issues on your fork?

Reply:
[They render "No response"](https://github.com/github/feedback/discussions/4491), so not sure if it is a great idea to include that many fields. We could always have two separate forms, though it would probably be a good idea to keep the number to a minimum.

Reply:
Thanks for the pointer :+1: I'll get back to this when I look at this issue template.